plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -354,11 +355,19 @@ func isAuthorizedForPolicy(user, sa user.Info, namespace, policyName string, aut
 }
 
 // authorizedForPolicy returns true if info is authorized to perform the "use" verb on the policy resource.
+// TODO: check against only the policy group when PSP will be completely moved out of the extensions
 func authorizedForPolicy(info user.Info, namespace string, policyName string, authz authorizer.Authorizer) bool {
+	// Check against extensions API group first (for backward compatibility)
+	return authorizedForPolicyInAPIGroup(info, namespace, policyName, extensions.GroupName, authz) ||
```

Comment:
I'd actually flip this, check the new desired API group first, then fall back to the extensions API group. We definitely want to announce deprecation and move all the addons to the policy API group in 1.11, so we can remove this double check asap.